### ISSUE TITLE
V2: playback issue

### ISSUE THREAD

**Issue by arnaud-neny** [farbrausch/fr_public#101]
Is there a way to get the proper playback of some older v2m files such as fr08 main? It's not playing like in the intro or the musidisk (and the issue is also happening in the winamp release).

**Comment by zvezdochiot** [comment]
Hi @MrArnal .

See also https://github.com/jgilje/v2m-player

**Comment by arnaud-neny** [comment]
yep, it's not playing properly some old v2m files. Building the v2 is not the pb. I even made an asm wrapper to be able to use the asm version on x64 (which plays better some v2m such as "We have accidently borrowed your votedisk".

**Comment by arnaud-neny** [comment]
I've got some issues also when changing the sample rate. The module "The scene is dead by Dubmood" doesn't play properly at 48KHz.
